linkml_runtime/linkml_model/linkml_files.py

```diff
@@ -152,7 +152,7 @@ def LOCAL_PATH_FOR(source: Source, fmt: Format) -> str:
 
 
 def GITHUB_IO_PATH_FOR(source: Source, fmt: Format, version="latest") -> str:
-    path = '/'.join([version, 'linkml_model', *_build_path(source, fmt)])
+    path = PurePath(version, 'linkml_model', *_build_path(source, fmt)).as_posix()
```

Comment:
Copying from #359 for the sake of recordkeeping:

This is for a URL, not an OS path, which is why it joins on / rather than using pathlib. Looks like the pathlib impl should be equivalent but just explaining the manual string munging

Reply:
Looks like the actual source of the bug for these is this line: https://github.com/linkml/linkml-runtime/blob/cf9e94b4b574e52dce2589645f6c514055b4d294/linkml_runtime/linkml_model/linkml_files.py#L87

Probably worth fixing that, but this works too.

Reply:
I will change this as you proposed but currently github has problems accepting pushes.